public/javascripts/Progress/src/Progress.js

```diff
@@ -76,5 +80,33 @@ function Progress (_, $, userRole) {
         });
     }
 
-    $('.put-user-org').on('click', putUserOrg);
+    // function to call endpoint and create team
+    function createTeam() {
+        escapeHTML
```

Comment:
Wait I assume that this line does nothing, right?

Reply:
yeah, that line was just moved downwards in the final its still there!

Reply:
Not sure what you mean by that... I'm referring to the first line of this func, the line that's just:
```
escapeHTML
```

Seems like it's just the function name. It's not calling the function or anything as far as I can tell?

Reply:
Yes absolutely right! That line should not be there (does nothing). Removed! Sorry didn't know what you were referencing at first.